examples/01_template/public/404.html

```diff
@@ -0,0 +1 @@
+<h1>Custom Not Found Page</h1>
```

Comment:
What if I want use javascript here?

Reply:
This looks like a manual setup not related to Waku itself

Reply:
Think it like a static image asset file. Up until now, it's very hard coded. So, this at least makes it configurable. And, it works without SSR.

I plan to do full customization in #329. But, if we want a solution "with React" we need another solution. I'll think about it, but it requires SSR.

A question for you: How should we deal with customizing 404 without SSR? What's the typical way with Vite + React SPA?

> not related to Waku itself

Yeah, it's not related to RSC. But, we need do something and this is a minimal setup.